### PR TITLE
first part of homework

### PR DESCRIPTION
**Pull request by LiluUncharted**
I made it from the beginning! 

image

### CODE REVIEW

**Comment by IvanCeban** [comment]
No need to request a code review if the feature is not finished. ;)
Let's finish it first

**Comment by LiluUncharted** [comment]
> No need to request a code review if the feature is not finished. ;)
> Let's finish it first


Ok 😁
But I don’t know how to finish it. )

**Comment by IvanCeban** [comment]
We will discuss this today and you still have time till Friday. So no worries

File: index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+    <link href="styles.css" rel="stylesheet">
```
Comment (sdmnc) [inline index.html]: Please move it to **head** tag

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+                text-align: center;
+                font-size: 1.3em;
+                font-weight: 500;
+                line-height: 30px;
+                letter-spacing: 0.2px;
+                color: #fff;
+                margin: 0px;
+            }
+
+            .button-get{
```
Comment (IvanCeban) [inline styles.css]: YOu have 2 similar buttons. Only colors are different.
I highly recommend you give them 2 classes to each. Ex: "bottom button-get" nad "button button-learn" 
Give all common styles to "button" class and use custom class for custom styles.

This advice works for all cases where you have similar elements with small amount of differences

**Review by IvanCeban — APPROVED** [review]
Great job!
Very good code styling
Good HTML layout

**Review by sdmnc — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (LiluUncharted) [reply index.html]: thanks, I didn't even notice it!

**Pull request merged**